src/label_studio_sdk/converter/converter.py

```diff
@@ -735,6 +743,90 @@ def add_image(images, width, height, image_id, image_path):
                 indent=2,
             )
 
+    def convert_to_brush(
```

Comment:
We have a separate file for brush functions - brush.py - please move it there.

Reply:
Moved convert_to_brush to brush.py in the PR repo with commit bd7bbba3b8ddb6295730aaae2261a2c28ba332ab